features/sanitizer.yml

```
@@ -1,5 +1,5 @@
 name: Sanitizer API
-description: The `parseHTML()` method for the `Document` interface and the `setHTML()` methods for the `Element` and `ShadowRoot` interfaces parse and insert HTML into the DOM in a way that can prevent cross-site scripting attacks. The `Sanitizer` API can customize the sanitization process.
+description: The `Document.parseHTML()` static method and the `setHTML()` method of `Element` and `ShadowRoot` parses and insert HTML into the DOM in a way that can prevent cross-site scripting attacks. The `Sanitizer` API can customize the sanitization process.
```

Something about this read a little off to me. In contrast to `Document`, a developer is unlikely to actually touch the `Element` and `ShadowRoot` interfaces directly, but manipulate instances. What do you think of this change?

```suggestion
description: The `Document.parseHTML()` static method and the `setHTML()` method of `Element` and `ShadowRoot` objects parse and insert HTML into the DOM in a way that can prevent cross-site scripting attacks. The `Sanitizer` API can customize the sanitization process.
```